plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -747,6 +747,27 @@ public void testLikeWithEscapePredicatePushdown()
         }
     }
 
+    @Test
+    public void testIsNullPredicatePushdown()
+    {
+        assertThat(query("SELECT nationkey FROM nation WHERE name IS NULL"))
```

Comment:
Let's also test with a column that actually contains NULLs.

Reply:
Also `IS NULL` is possible to represent as a TupleDomain (`none`) so should we add an `OR` to ensure expression pushdown path?